src/components/TransactionHistoryRows/TransactionGenericRow.tsx

```diff
@@ -216,31 +212,27 @@ export const TransactionGenericRow = ({
         </Flex>
         {isLargerThanLg && (
           <Flex alignItems='flex-start' flex={1} flexDir='column' textAlign='right'>
-            {fee && bnOrZero(fee?.amount).gt(0) ? (
+            {fee && bnOrZero(fee.value).gt(0) && (
```

Comment:
ternary

Reply:
As discussed yesterday, fine in this case, as short-circuiting on `fee` would evaluate to `undefined`, and short-circuiting on `bnOrZero(fee.value).gt(0)`would evaluate to false, both of them react will understand as "should not render anything"

https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored